src/OpenApi.Extensions/OpenApiExtensions.cs

```diff
@@ -171,7 +171,10 @@ contexts.Count is 1 ?
                 foreach (var assembly in assemblies)
                 {
                     var documentation = new AddXmlDocumentationTransformer(assembly);
```

Comment:
If the implementations are left split, then the existing implementation should be renamed as it's currently named as if it's **the** XML documentation transformer, which it isn't if there's two.